components/molecule/select/src/index.js

```diff
@@ -125,7 +127,10 @@ const MoleculeSelect = forwardRef((props, forwardedRef) => {
     }
   }
 
-  const handleFocusOut = () => setFocus(false)
+  const handleFocusOut = e => {
```

Comment:
plz, be explicit on variable naming:
```suggestion
  const handleFocusOut = event => {
```

Reply:
good catch, updated!